Data generation process:

## SRR7366586.sub.1.fq.gz and SRR7366586.sub.2.fq.gz

SRR7366569.1.fq.gz SRR7366569.2.fq.gz generated by nf-hlatyping, subset bam files were converted to fastq files for realignment with bwa-mem for hla-typing with Polysolver.

These were subset with the following commands:

```
singularity pull docker://biocontainers/seqtk:v1.3-1-deb_cv1
seqtk sample -s100 SRR7366586.1.fq.gz 370000 > SRR7366586.sub.1.fq
seqtk sample -s100 SRR7366586.2.fq.gz 370000 > SRR7366586.sub.2.fq
gzip SRR7366586.sub.1.fq
gzip SRR7366586.sub.2.fq
```

The number of reads was chosen such that the size of the output files was less than 20MB.

## chr6-subset.fa

This is a subset of the GRCh38 reference genome, chr6:29500000-33200000. It was generated with the following command:
Dowloaded chr6 from UCSC: 

```
wget https://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/chr6.fa.gz
gunzip chr6.fa.gz
echo "chr6       29500000        33200000" > chr6-region.bed
singularity pull docker://biocontainers/bedtools:v2.27.1dfsg-4-deb_cv1
singularity shell bedtools_v2.27.1dfsg-4-deb_cv1.sif
bedtools getfasta -fi chr6.fa -bed chr6-region.bed -fo chr6-subset.fa
```

## example_pe.1.fq.gz example_pe.2.fq.gz

Generated from example_pe.bam from nf-core/test-datasets as the paired-end fastq files had differing numbers of lines.

Converted to fastq using the following:


```
singularity run /home/kryan/misc/useful/nextflow/nf-hlatyping/testdata/bam/biocontainers-samtools-v1.9-4-deb_cv1_dev.sif ./run.sh
```

`run.sh`

```
samtools sort -l 9 -T tmp -@ 4 -n *.bam -o - |
    samtools fastq -c 6 /dev/stdin \
    -1 "example_pe.1.fq.gz" \
    -2 "example_pe.2.fq.gz" \
    -0 /dev/null -s /dev/null -n
```


